packages/wallet/src/providers/polkadot/polkadot-provider.ts

```diff
@@ -48,12 +58,52 @@ export class PolkadotProvider extends EventBus<ExtensionProviderEvents> {
     const opts = options({
       provider: wsProvider,
     });
+    // eslint-disable-next-line no-async-promise-executor
+    await new Promise(async (resolve, reject) => {
+      wsProvider.on('connected', () => {
+        resolve();
+      });
+      let reported = false;
+      wsProvider.on('error', (e) => {
+        if (reported) {
+          return;
+        }
+        let interActiveFeedback: InterActiveFeedback;
+        const body = InterActiveFeedback.feedbackEntries([
+          {
+            header: 'Failed to establish WS connection',
```

Comment:
we should probably abstract all our error messages into a common folder/location as well and use Enums.

Reply:
Added implementation for the Error here https://github.com/webb-tools/webb-dapp/blob/c936d7541f44e5216b6f833790710993deb39843/packages/react-components/src/InteractiveFeedbackView/InteractiveErrorView.tsx#,
and as you suggested we can handle the  content for error messages